chromadb/cli/cli.py

```diff
@@ -48,7 +51,8 @@ def run(
     print("Running Chroma")
     print("\033[0m")  # Reset
 
-    typer.echo(f"\033[1mSaving data to\033[0m: \033[32m{path}\033[0m")
+    if persistent:
```

Comment:
It would make sense to add a print statement letting the user know they are running in in-memory mode and that data does not persist after the application exits.

Reply:
Good idea, i will add it.